docs/topics/feed-exports.rst

```diff
@@ -227,6 +241,26 @@ The feeds are written to the standard output of the Scrapy process.
  * Required external libraries: none
 
 
+.. _delayed-file-delivery:
+
+Delayed file delivery
+---------------------
+
+As indicated above, some of the described storage backeds use delayed file
+delivery.
+
+These storage backeds do not receive items as they are scraped. Instead, Scrapy
```

Comment:
This is slighhtly ambiguous: I'd consider storage backend to be a Python class which implements delivery to a particular location. These Python classes do receive items as thy are scraped.

(there is also a small typo - backeds instead of backends)